net-utils/src/lib.rs

```diff
@@ -867,15 +877,56 @@ mod tests {
             bind_common_in_range_with_config(ip_addr, (3200, 3250), config).unwrap();
 
         let rt = runtime();
-        assert!(!rt.block_on(do_verify_reachable_ports(
+        assert!(!rt.block_on(verify_reachable_udp_ports(
             server_ip_echo_addr,
-            vec![],
             &[&client_udp_socket],
             Duration::from_secs(2),
             3,
         )));
     }
 
+    #[test]
+    fn test_verify_many_ports_reachable() {
+        solana_logger::setup();
```

Comment:
do we really need to write logs in the tests?

Reply:
It helps with debugging when tests fail. all tests here have them.

Reply:
I think it is generally wrong to log in the unit tests because if test fails from time to time than it should be debug/fixed independently on the CI runs. The fact that we have these log lines there suggests that some of these tests may rely on some implicit observations rather than explicit assertions.
But I understand that you don't want to fix all things in one shot

Reply:
I can make a separate PR for this. A deeper look into the usage of this code (and whether we can simplify things) would be justified too.